dts/bindings/sensor/invensense,icm42670.yaml

```diff
@@ -64,6 +81,40 @@ properties:
       - 4
       - 2
 
+  accel-avg:
+    type: int
+    default: 32
+    description: |
+      Averaging filter setting to create accelerometer output
+      in accelerometer low power mode.
+      Maps to ACCEL_UI_AVG field in ACCEL_CONFIG1 setting.
+      The default corresponds to the reset value of the register field.
+    enum:
+      - 2
+      - 4
+      - 8
+      - 16
+      - 32
+      - 64
+
+  accel-filt-bw:
+    type: int
+    default: 180
```

Comment:
units?

Reply:
Unit is Hz, I added it in the comment.

Reply:
property still misses proper suffix

Reply:
I don't get your point. Which suffix is missing for accel-filt-bw?